### DISCUSSION TITLE
Only log I explicilty put in code for.

### DISCUSSION THREAD

**Discussion by kglasnapp** [jonahsnider/doglog#70] [Q&A] [answered]
Can I set an option to only log the items I wrote code for. In the example below I only want the line with "/Robot/Dog:",500" i.e. the one I wrote a log statement for --> DogLog.log("Dog:", count);

5.572831,"/Robot/SystemStats/EpochTimeMicros",5572908
5.593117,"/Robot/DogLog/QueueRemainingCapacity",975
5.593049,"/Robot/SystemStats/EpochTimeMicros",5593123
5.604106,"/Robot/Dog:",500
5.612277,"/Robot/SystemStats/EpochTimeMicros",5612332
5.632763,"/Robot/DogLog/QueuedLogs",31
5.632763,"/Robot/DogLog/QueueRemainingCapacity",969

**Comment by jonahsnider** [comment, marked as answer]
Hey! Thanks for trying out DogLog. The extra logs you're seeing there are various system stats which can be disabled by setting the "log extras" setting to `false:

```java
DogLog.setOptions(new DogLogOptions().withLogExtras(false));
```

You can read more about that option on the documentation [here](https://doglog.dev/reference/configuring/#log-extras).

The internal logs under the `/Robot/Doglog/` namespace can't be disabled, those will always be recorded. Let me know if you have any additional questions!
